junit-reporting-handlebars/src/test/java/com/github/bogdanlivadariu/reporting/junit/builder/AllJunitReportsTest.java

```diff
@@ -12,13 +12,13 @@
 import org.junit.Test;
 
 public class AllJunitReportsTest {
-    List<String> xmlReports;
+    private List<String> xmlReports;
 
-    String reportPath = this.getClass().getClassLoader().getResource("valid-report-1.xml").getPath();
+    private String reportPath = this.getClass().getClassLoader().getResource("valid-report-1.xml").getPath();
 
-    JUnitReportBuilder builder;
+    private JUnitReportBuilder builder;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'builder' could be replaced by a local variable.](https://www.codacy.com/app/bogdan-livadariu/bootstraped-multi-test-results-report/file/2703449482/issues/source?bid=3293946&fileBranchId=3342494#l19)